doc/api-extensions.md

```diff
@@ -2618,3 +2618,7 @@ Note that the `openid` and `email` scopes are always required.
 ## `project_default_network_and_storage`
 
 Adds flags --network and --storage. The --network flag adds a network device connected to the specified network to the default profile. The --storage flag adds a root disk device using the specified storage pool to the default profile.
+
+## `bpf_delegation`
```

Comment:
As this is container specific, suggest `container_bpf_delegation`, similar to `container_syscall_intercept_bpf_devices` that exists today.

Reply:
Fixed, thanks!